### PR TITLE
feat: loosen description requirements for toggle component

### PR DESCRIPTION
**Pull request by zsoh97**

### CODE REVIEW

File: react/src/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export interface ToggleProps extends SwitchProps {
   /**
    * Secondary description text
    */
-  description?: string
+  description?: string | Element
```
Comment (karrui) [inline react/src/Toggle/Toggle.tsx:22]: i think can loosen to `ReactNode`!

Comment (zsoh97) [reply react/src/Toggle/Toggle.tsx:22]: yea, my bad hahahaha. i didnt even npm ci . will update

File: react/src/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export interface ToggleProps extends SwitchProps {
   /**
    * Secondary description text
    */
-  description?: string
+  description?: string | ReactNode
```
Comment (karrui) [inline react/src/Toggle/Toggle.tsx:22]: nit: ReactNode consists of `string`

```suggestion
  description?: ReactNode
```

**Review by karrui — CHANGES_REQUESTED** [review]

File: react/src/Toggle/Toggle.tsx (hunk 2 above)
Comment (zsoh97) [reply react/src/Toggle/Toggle.tsx]: lol my bad, thanks for flagging! fixed

**Review by karrui — APPROVED** [review]
lgtm

**Pull request merged**